Providers/Scripts/2.6x-2.7x/Scripts/Tests/test_nxProviders.py

```diff
@@ -1998,6 +1998,18 @@ def testInventoryMarshall(self):
         self.assertTrue(r[0] == 0,"Inventory_Marshall('*', " + self.provider + "provider, False,'')  should return == 0")
         print repr(r[1])
 
+    def testInventoryMarshallControllerWildcard(self):
+        r=nxService.Inventory_Marshall('*', '*', False,'')
+        self.assertTrue(r[0] == 0,"Inventory_Marshall('*', " + self.provider + "provider, False,'')  should return == 0")
+        print repr(r[1])
+
+    def testInventoryMarshallControllerError(self):
+        l = ['systemd', 'upstart', 'init']
+        l.remove(self.provider)
+        r=nxService.Inventory_Marshall('*', l[0], False,'')
```

Comment:
You can also test that Inventory_Marshall with `l[1]` fails to be more thorough.
Can you use descriptive variable names like `providers` or `controllers` instead of `l`?